api/cluster/templater.go

```diff
@@ -160,6 +161,21 @@ func createPredictorSpec(modelService *models.Service, config *config.Deployment
 					Container: corev1.Container{
 						Name:      kserveconstant.InferenceServiceContainerName,
 						Resources: resources,
+						LivenessProbe: &corev1.Probe{
+							Handler: corev1.Handler{
+								HTTPGet: &corev1.HTTPGetAction{
+									Path:   fmt.Sprintf("/v1/models/%s", modelService.Name),
+									Scheme: "HTTP",
+									Port: intstr.IntOrString{
+										IntVal: 8080,
+									},
+								},
+							},
+							InitialDelaySeconds: 10,
```

Comment:
Can you add `failureThreshold` ? And also can you extract all these values into a constant to avoid repetition?